chapters/install/os/via-pip.rst

```diff
@@ -87,15 +85,15 @@ However, some Executors may require extra dependencies. The full table of these
 
 .. code-block:: bash
 
-   pip install "jina[nlp+cv]"
+   pip install -U "jina[nlp+cv]"
```

Comment:
I think this part is more about installation instead of updating.

Reply:
You are correct, but it's what Han had in the readme?